pubsub/cloud-client/publisher.py

```diff
@@ -109,6 +110,36 @@ def publish_messages_with_futures(project, topic_name):
         print(future.result())
 
 
+def publish_messages_with_error_handler(project, topic_name):
```

Comment:
Please add region tags. See the work Alix is doing in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1491

Reply:
Hi Tim, most Python doc uses indented_block instead of region_tag to embed code snippets. Should I still add region tags?

Reply:
Yes, please. It's required for go/samples-tracker to work.

Reply:
Region tag added. Sorry for the late response :(